Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -439,6 +448,38 @@ void doPublishRawImu() {
 }
 
 /*
+ * Publish bumper data to ros.
+ */
+void doPublishBumpers(void) {
+  ros::Time current_time = ros_nh.now();
+
+  int rawBumperLeft = analogRead(BUMPER_LEFT_PIN);
+  int rawBumperRight = analogRead(BUMPER_RIGHT_PIN);
+
+  double bumperLeft = MAX_FORCE_LEFT_BUMPER 
+                        * ((rawBumperLeft - MIN_RAW_LEFT_BUMPER)
+                            /((double)(MAX_RAW_LEFT_BUMPER - MIN_RAW_LEFT_BUMPER)));
```

Comment:
It's preferable to use `static_cast<double>(MAX ...)`

To shorten the line width, you could 
```
static const int RANGE_RAW_LEFT_BUMPER(MAX_RAW_LEFT_BUMPER - MIN_RAW_LEFT_BUMPER);
```